content/en/tracing/guide/inferred-service-opt-in.md

```diff
@@ -0,0 +1,264 @@
+---
+title: New Service page and Inferred Services
+kind: Guide
+disable_toc: false
+private: true
+further_reading:
+- link: "/tracing/services/"
+  tag: "Documentation"
+  text: "Service Observability"
+- link: "/tracing/trace_collection/"
+  tag: "Documentation"
+  text: "Sending Traces to Datadog"
+- link: "/tracing/trace_collection/dd_libraries/"
+  tag: "Documentation"
+  text: "Add the Datadog Tracing Library"
+---
+
+{{< callout url="https://docs.google.com/forms/d/1imGm-4SfOPjwAr6fwgMgQe88mp4Y-n_zV0K3DcNW4UA/edit" d_target="#signupModal" btn_hidden="true" btn_hidden="false" header="Opt in to the private beta!" >}}
+Inferred services and the new Service Page layout are in private beta. To request access, complete the form.
+{{< /callout >}}
+
+Follow the steps below to enable the new dependency map on the [Service Page][1], and add _Inferred services_ to the [Service Map][2] and [Service Catalog][3] pages.
```

Comment:
The first two links did not work for me, see note at the bottom of page.

Reply:
Thanks, I think that was an errant copy-pasta